packages/react-app/src/components/views/quest-list.tsx

```diff
@@ -141,6 +141,7 @@ export default function QuestList() {
       if (newQuest) {
         setQuests([newQuest, ...quests]);
         setNewQuestLoading(false);
+        refresh(filter);
```

Comment:
I dont like that much the whole refresh, I would rather go with a full check of the filter like this

 if (
          (filter.status === ENUM_QUEST_STATE.All || newQuest.state === filter.status) &&
          filter.title &&
          filter.title.includes(newQuest.title!) &&
          filter.description &&
          filter.description.includes(newQuest.description!) &&
          filter.minExpireTime &&
          filter.minExpireTime <= newQuest.expireTime
        ) {
          setQuests([newQuest, ...quests]);
        }